.editorconfig

```diff
@@ -0,0 +1,8 @@
+# Top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = false
```

Comment:
I think final newlines should always be included?

Reply:
I'm not opposed.